README.md

```diff
@@ -1,5 +1,7 @@
 # README
 
+# This is a Microsoft Demo
```

Comment:
This is a good change, i like it

Reply:
@pear-on can you explain this diff?

Reply:
Sure, the diff shows that two lines have been added to the README file. `+# This is a Microsoft Demo` is a comment indicating that the code is part of a Microsoft demo. If you have any other questions, let me know!

Reply:
@pear-on what is this commit?

Reply:
@pear-on what is this commit?

Reply:
This commit adds a demo comment to the README for demonstration purposes.